docs/getting-started-guides/kubeadm.md

```diff
@@ -213,199 +278,298 @@ kube-system   canal-node-77d0h                  2/3       CrashLoopBackOff    3
+
+```
+  2m		2m		1	{kubelet nac}	spec.containers{flannel}		Warning		Failed		Failed to start container with docker id 927e7ccdc32b with error: Error response from daemon: {"message":"chown /etc/resolv.conf: operation not permitted"}
 ```
 
-Or
+Or this:
+
 ```
   6m	1m	191	{kubelet nac}		Warning	FailedSync	Error syncing pod, skipping: failed to "SetupNetwork" for "kube-dns-2924299975-1l2t7_kube-system" with SetupNetworkError: "Failed to setup network for pod \"kube-dns-2924299975-1l2t7_kube-system(dee8ef21-fbcb-11e6-ba19-38d547e0006a)\" using network plugins \"cni\": open /run/flannel/subnet.env: no such file or directory; Skipping pod"
 ```
 
-You can then do some Google searches on the error messages, which may help you to find some solutions.
+You can then do some Google searches on the error messages, which may help you
+to find some solutions.
 
 ### (4/4) Joining your nodes
 
-The nodes are where your workloads (containers and pods, etc) run.
-If you want to add any new machines as nodes to your cluster, for each machine: SSH to that machine, become root (e.g. `sudo su -`) and run the command that was output by `kubeadm init`.
-For example:
-
-    # kubeadm join --token <token> <master-ip>
-    [kubeadm] WARNING: kubeadm is in alpha, please do not use it for production clusters.
-    [preflight] Running pre-flight checks
-    [preflight] Starting the kubelet service
-    [tokens] Validating provided token
-    [discovery] Created cluster info discovery client, requesting info from "http://192.168.x.y:9898/cluster-info/v1/?token-id=f11877"
-    [discovery] Cluster info object received, verifying signature using given token
-    [discovery] Cluster info signature and contents are valid, will use API endpoints [https://192.168.x.y:6443]
-    [bootstrap] Trying to connect to endpoint https://192.168.x.y:6443
-    [bootstrap] Detected server version: v1.5.1
-    [bootstrap] Successfully established connection with endpoint "https://192.168.x.y:6443"
-    [csr] Created API client to obtain unique certificate for this node, generating keys and certificate signing request
-    [csr] Received signed certificate from the API server:
-    Issuer: CN=kubernetes | Subject: CN=system:node:yournode | CA: false
-    Not before: 2016-12-15 19:44:00 +0000 UTC Not After: 2017-12-15 19:44:00 +0000 UTC
-    [csr] Generating kubelet configuration
-    [kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/kubelet.conf"
-
-    Node join complete:
-    * Certificate signing request sent to master and response
-      received.
-    * Kubelet informed of new secure connection details.
-
-    Run 'kubectl get nodes' on the master to see this machine join.
-
-A few seconds later, you should notice that running `kubectl get nodes` on the master shows a cluster with as many machines as you created.
+The nodes are where your workloads (containers and pods, etc) run. If you want
+to add any new machines as nodes to your cluster, for each machine: SSH to that
```

Comment:
"To add any new machines to your cluster as nodes, you must do the following for each machine"

Reformat the instructions as a numbered list.